composer.json

```diff
@@ -14,7 +14,7 @@
         "phpunit/phpunit": "^9.5",
         "rector/phpstan-rules": "^0.6",
         "rector/rector-debugging": "dev-main",
-        "rector/rector-src": "dev-main",
+        "rector/rector-src": "dev-main#377713ae7044739b33a2186bbebbd253f7a0efdd",
```

Comment:
this is needed to pick last commit at PR:

- https://github.com/rectorphp/rector-src/pull/3088